_events/seattle-local-meeting.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Seattle Local Tech Worker Meetup"
+date: 2024-06-18 18:30:00
```

Comment:
Currently the preview renders this as "11:30 PDT" when you wanted 18:30 PDT

There are two issues here.

1. Seconds should never be included, because jekyll tries to parse this as a Timezone override and is confused.
2. The site wide time zone is set to EST (New York) so there are two correct ways to write this, either as

`2024-06-18 15:30` which relies on time zone set in site wide config https://github.com/techworkersco/twc-site/blob/master/_config.yml#L22 or `2024-06-18 18:30 -0800` which explicitly includes offset. We cannot rely on "time_zones" variable, because that could display multiple time zones in the event of an online event for example.